src/h2/create-table.sql

```diff
@@ -2,23 +2,21 @@
 -- You may use, distribute and modify this code under the
 -- terms of the Apache License v2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt).
 
-create table BOUNDING_BOX(
+create table if not exists BOUNDING_BOX(
   file_name varchar(255) primary key,
   top int not null,
   left int not null,
   bottom int not null,
   right int not null,
   width int not null,
-  height int not null
+  height int not null,
+  dataset varchar(255) default 'UNKNOWN'
```

Comment:
It's not even for simplicity. More so that you can't insert values that are not defined as part of the enum - which is a very nice plus, i.e., you'll have to do that otherwise in your code so.

But yeah, fair enough in limiting the changes. If you're gonna change this to another RDS, unless there specific requirements you'd need from it, sticking with an AWS hosted Postgres/MySql should be fine I guess.